test/runtests.jl

```diff
@@ -0,0 +1,84 @@
+using SparseArrays
+using LinearAlgebra
+using BlockNLPModels
+using BlockNLPAlgorithms
+using JuMP
+using NLPModelsJuMP
+using Test
+
```

Comment:
Can we add a comment describing the shape of the optimization problem we are solving here?

Reply:
(I assume it's a LQR)